lua/org-roam/setup/keybindings.lua

```diff
@@ -82,62 +87,65 @@ end
 local function assign_core_keybindings(roam)
     -- User can remove all bindings by setting this to nil
     local bindings = roam.config.bindings or {}
+    local prefix = roam.config.bindings.prefix or "<Leader>n"
```

Comment:
You shouldn't need a default here. It's assumed that the prefix will be present in some form, since we provide a default value at the config level.

Reply:
I know, it's redundant. I will remove it.